frontend/packages/kubevirt-plugin/integration-tests/tests/cnv-installation.scenario.ts

```diff
@@ -0,0 +1,25 @@
+import * as sidenavView from '@console/internal-integration-tests/views/sidenav.view';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { $, browser, by, element } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+
+const namespaceButton = $('.co-namespace-selector button');
+const openshiftNamespaceButton = $('#openshift-cnv-link');
```

Comment:
it's `openshiftCNV` namespace

Reply:
Can we move these variables to a view file?

Reply:
> it's `openshiftCNV` namespace

for this var "openshiftNamespaceButton"? (like to change it from "openshift-cnv-link" to "openshiftCNV"?)

Reply:
moved